pkg/vm/vm.go

```diff
@@ -136,6 +136,12 @@ func (v *VM) SetGasLimit(max util.Fixed8) {
 	v.gasLimit = max
 }
 
+// AddGas consumes specified amount of gas.
```

Comment:
How can I know what it returns from this description?

Reply:
Fixed.